.vscode/tasks.json

```diff
@@ -67,6 +67,19 @@
         "problemMatcher": "$msCompile",
         "group": "build"
     },
+    {
+        "label": "build Roslyn.sln",
```

Comment:
I didn't follow what this change has to do with the PR

Reply:
It's adding a build entry for VS code so that you can build Roslyn.sln. Was important to making sure I had tracked down all of the necessary changes.